project-base/tests/FrontendApiBundle/Functional/Order/GetOrderAsAuthenticatedCustomerUserTest.php

```diff
@@ -52,14 +53,15 @@ public function testGetOrderReturnsError(): void
      */
     private function getOrderDataForCurrentlyLoggedCustomerUserProvider(): array
     {
+        $firstDomainLocale = $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getLocale();
         $data = [];
         $orderIds = [1, 2, 3];
         foreach ($orderIds as $orderId) {
             $order = $this->orderFacade->getById($orderId);
             $data[] = [
                 $order->getUuid(),
                 [
-                    'status' => $order->getStatus()->getName(),
+                    'status' => $order->getStatus()->getName($firstDomainLocale),
```

Comment:
This should not be necessary as tests are run always on first domain and its locale is set as default in translation entities